lib/src/xml.rs

```diff
@@ -220,16 +208,39 @@ impl TransformerTrait for Transformer {
                                     } else {
                                         println!("Error: No value");
                                     }
-                                }
+                                },
+                                "align" => {
+                                    if let Some(value) = &child.text {
+                                        align = value.to_string();
+                                    } else {
+                                        println!("Error: No value");
```

Comment:
Could you remove all of println?

Add "log = "0.4.20" to Cargo.toml please

Probably better way to inform about problems should look like this:

log::warn!("No align value for image");

Reply:
I thought I could, but it's not just replace, I tried running tests and it didn't print... Let's open an issue to insert logs and remove all prints, so I can work more calmly on this.